packages/base-flambda/base-flambda.base/opam

```diff
@@ -0,0 +1,5 @@
+opam-version: "1.2"
+maintainer: "stephen.dolan@cl.cam.ac.uk"
+build: [["ocaml" "check_flambda.ml"]]
+available: [ ocaml-version >= "4.03" ]
```

Comment:
(i.e. by using `compiler` constraints rather than `ocaml-version` constraints in the `available` field)